crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -214,6 +215,80 @@ impl InfinitePlane3d {
 
         (Self { normal }, translation)
     }
+
+    /// Computes the shortest distance between a plane transformed with the given `isometry` and a
+    /// `point`.
+    #[inline]
+    pub fn signed_distance(&self, isometry: Isometry3d, point: Vec3) -> f32 {
+        (isometry * self.normal).dot(isometry * Vec3::ZERO - point)
```

Comment:
Haven't thought this through much yet, but could this be done equivalently (and perhaps faster) as:
`-self.normal.dot(inverse_isometry * point)` ? (The idea is to transform the point to the plane rather than the other way around / use the fact that isometries preserve dot products. (although that part is a bit more nuanced here given that Dir3 has different transformation semantics than the Vec3)).

Reply:
Yeah that's actually true. Thanks! I was a bit concerned about the inverse causing a performance hit, but looking into the implementation it should be fine. I'll change that 👍🏼